main.py

```diff
@@ -1,63 +1,11 @@
-from fastapi import FastAPI, HTTPException
-from pydantic import BaseModel 
+from fastapi import FastAPI
 import models 
-from sqlalchemy.orm import Session
-from fastapi import Depends
-from database import engine, get_db
+from database import engine
 import uvicorn
 
-
 models.Base.metadata.create_all(bind=engine)
 
 app = FastAPI()
 
```

Comment:
You should include the routers you created using: app.include_router(name, tags)